cloudmarker/baseconfig.py

```diff
@@ -86,3 +104,7 @@
       - file_handler
 
 schedule: "00:00"
+"""
+
+
+config_dict = yaml.load(config_yaml)
```

Comment:
I like the current approach because of three reasons:

- By using `config_yaml` string as the original source of base config, the `--print-base-config` option can print the YAML keys in the order we intend. `yaml.dump(config_dict)` may print the keys in any arbitrary order which may not aid comprehensibility.
- We can put comments in the `config_yaml` string and they would appear intact with the `--print-base-config` option.
- Any custom formatting (such as blank lines etc.) we use to aid readability would appear intact with the `--print-base-config` option. This is same as the point you mentioned.